### PR TITLE
0.5

### PR DESCRIPTION
**Pull request by marcusklaas**
Prepares for a 0.5 release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,7 +124,20 @@ By default, the binary is built as well. If you don't want/need it, then build l
...
 ```toml
-pulldown-cmark = { version = "0.4", default-features = false }
+pulldown-cmark = { version = "0.5", default-features = false }
+```
+
+SIMD accelerated scanners are available for the x64 platform from version 0.5 onwards. To
+enable them, build with simd feature:
+
+```bash
+> cargo build --features simd
```
Comment (raphlinus) [inline README.md]: I think it might make sense to add `--release` here, a debug build with SIMD isn't very helpful.

File: src/html.rs [hunk 2]
```diff
@@ -409,6 +409,11 @@ where
 /// Iterate over an `Iterator` of `Event`s, generate HTML for each `Event`, and
 /// write it out to a writable stream.
 ///
+/// **Note**: using this function with an unbuffered writer like a file or socket
+/// will result in poor performance. Wrap these in a
+/// [`Bufwriter`](https://doc.rust-lang.org/std/io/struct.BufWriter.html) to
```
Comment (raphlinus) [inline src/html.rs]: BufWriter (capitalization)

File: README.md [hunk 3]
```diff
@@ -119,7 +124,20 @@ By default, the binary is built as well. If you don't want/need it, then build l
...
+```
+
+Or add the feature to your project's `Cargo.toml`:
+
+```toml
+pulldown-cmark = { version = "0.5", default-features = false, features = ["simd"] }
```
Comment (raphlinus) [inline README.md]: I wonder if we can remove the getopts default feature from library builds while leaving them in for binaries (the current state was added in #56), but I don't see an obvious way to do it, at least without having two separate crates, one for the lib and one for the bin. This is probably a pretty minor nit and shouldn't block this release.

Comment (marcusklaas) [reply README.md]: Hmmm yes, that would be nice. I don't see how we could do it either though…

**Pull request merged**